### PR TITLE
Allow mets:file without GROUPID

### PR DESCRIPTION
**Pull request by sevein**
I've seen this happening in transfer METS.

### CODE REVIEW

File: fixtures/mets_without_groupid_in_file.xml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    </mets:div>
+  </mets:structMap>
+  <mets:structMap LABEL="processed" TYPE="physical">
+    <mets:div TYPE="directory" LABEL="Prueba007-44db0a40-4c76-45b9-83f1-a8adba434b43">
+      <mets:div TYPE="directory" LABEL="objects">
+        <mets:fptr FILEID="file-db653873-d0ab-4bc1-9edb-2b6d2d84ab5a"/>
+      </mets:div>
+    </mets:div>
+  </mets:structMap>
+</mets:mets>
```
Comment (mistydemeo) [inline fixtures/mets_without_groupid_in_file.xml]: Nitpick, but newline at end of file?

**Comment by sevein** [comment]
Done.

**Comment by Hwesta** [comment]
:+1:

File: metsrw/mets.py [hunk 2]
```diff
@@ -251,9 +251,11 @@ def _parse_tree_structmap(self, tree, parent_elem):
                 if file_elem is None:
                     raise exceptions.ParseError('%s exists in structMap but not fileSec' % file_id)
                 file_uuid = file_id.replace(utils.FILE_ID_PREFIX, '', 1)
-                group_uuid = file_elem.get('GROUPID').replace(utils.GROUP_ID_PREFIX, '', 1)
-                if group_uuid != file_uuid:
-                    derived_from = group_uuid  # Use group_uuid as placeholder
+                group_uuid = file_elem.get('GROUPID')
+                if group_uuid:
+                    group_uuid.replace(utils.GROUP_ID_PREFIX, '', 1)
```
Comment (Hwesta) [inline metsrw/mets.py]: Oops, missed this in code review - `.replace` doesn't modify in place, but returns the updated value.  Fix would be
```python
group_uuid = file_elem.get('GROUPID', '').replace(utils.GROUP_ID_PREFIX, '', 1)
    if group_uuid != file_uuid:
    derived_from = group_uuid  # Use group_uuid as placeholder
```
or wait for the changes in dev/reingest to come in.

**Pull request closed without merging**